install/0000_30_machine-api-operator_01_images.configmap.yaml

```diff
@@ -23,5 +23,6 @@ data:
       "clusterAPIControllerIBMCloud": "registry.svc.ci.openshift.org/openshift:ibmcloud-machine-controllers",
       "clusterAPIControllerOvirt": "registry.svc.ci.openshift.org/openshift:ovirt-machine-controllers",
       "clusterAPIControllerPowerVS": "registry.svc.ci.openshift.org/openshift:powervs-machine-controllers",
-      "clusterAPIControllerVSphere": "registry.svc.ci.openshift.org/openshift:machine-api-operator"
+      "clusterAPIControllerVSphere": "registry.svc.ci.openshift.org/openshift:machine-api-operator",
+      "clusterAPIControllerNutanix": "public.ecr.aws/g7v6l0u0/openshift/mapi-provider-nutanix:latest"
```

Comment:
We probably should have a Openshift registry link for Nutanix MAPI provider too. I reckon we need to get https://github.com/nutanix-cloud-native/machine-api-provider-nutanix seeded over to the Openshift org.